BRAD/coder.py

```
@@ -180,16 +180,27 @@ def codeCaller(state):
-            "Completion Tokens": cb.completion_tokens,
-            "Total Cost (USD)": cb.total_cost
-        }        
+
+        try:
+            responseOriginal['call back'] = {
+                "Total Tokens": cb.total_tokens,
+                "Prompt Tokens": cb.prompt_tokens,
+                "Completion Tokens": cb.completion_tokens,
+                "Total Cost (USD)": cb.total_cost
+            }
+        except:
```

Ruff (0.7.0)

Do not use bare `except`

(E722)